Source/Charts/Renderers/LineChartRenderer.swift

```diff
@@ -32,19 +32,16 @@ open class LineChartRenderer: LineRadarRenderer
     {
         guard let lineData = dataProvider?.lineData else { return }
         
-        for i in 0 ..< lineData.dataSetCount
+        for i in lineData.indices
         {
-            guard let set = lineData.getDataSetByIndex(i) else { continue }
-            
-            if set.isVisible
+            guard let set = lineData[i] as? LineChartDataSetProtocol else
             {
-                if !(set is LineChartDataSetProtocol)
-                {
-                    fatalError("Datasets for LineChartRenderer must conform to LineChartDataSetProtocol")
-                }
-                
-                drawDataSet(context: context, dataSet: set as! LineChartDataSetProtocol)
+                fatalError("Datasets for LineChartRenderer must conform to LineChartDataSetProtocol")
             }
+
+            guard set.isVisible else { continue }
```

Comment:
I'm thinking in this case,
```swift
if set.isVisible
{
    drawDataSet(context: context, dataSet: set)
}
```
is a little bit nicer? using guard here seems like a over kill. What you think